DQM/Physics/src/TopSingleLeptonDQM_miniAOD.cc

```diff
@@ -373,6 +379,8 @@ void MonitorEnsemble::fill(const edm::Event& event,
   // fill monitoring plots for primary verices
   edm::Handle<edm::View<reco::Vertex> > pvs;
   if (!event.getByToken(pvs_, pvs)) return;
+  const reco::Vertex& PV= pvs->front();
```

Comment:
please start variables with lower case (and in any case avoid variables that differ only by their case)